shell/platform/tizen/tizen_input_method_context.h

```diff
@@ -69,7 +73,8 @@ class TizenInputMethodContext {
   void SetContextOptions();
   void SetInputPanelOptions();
 
-  TizenWindow* window_ = nullptr;
+  bool ShouldNotFilterEvent(std::string key, bool is_ime);
```

Comment:
I picked `ShouldIgnoreKey`

Reply:
Do you want to keep the name `HandleUnfilteredEvent` as it is because that part will be reworked soon? https://github.com/flutter-tizen/engine/pull/291#discussion_r883870830

Reply:
Thank you for the detailed review, I changed it. please review again.

Reply:
Okay, thanks!